qa/L0_response_cache/test.sh

```diff
@@ -29,12 +29,54 @@ RET=0
 
 TEST_LOG="./response_cache_test.log"
 UNIT_TEST=./response_cache_test
+export CUDA_VISIBLE_DEVICES=0
+
+# Only localhost supported in this test for now, but in future could make
+# use of a persistent remote redis server, or similarly use --replicaof arg.
+export TRITON_REDIS_HOST="localhost"
+export TRITON_REDIS_PORT="6379"
 
 rm -fr *.log
 
-# UNIT TEST
+function install_redis() {
+  ## Install redis if not already installed
```

Comment:
Why not install redis directly in our qa container? It would help keeping all the test dependencies in one place.

Reply:
I'd like to keep it local to the test for now to avoid bloating the container - I think it gets harder to track which packages belong to which tests, for cleaning up unused packages in the future. If redis expands to other tests in the future, I would probably move it to QA container install at that point.

Reply:
> I'd like to keep it local to the test for now to avoid bloating the container ... If redis expands to other tests in the future

If the test should be self-contained and only testing redis specific functionality, should it be moved to redis cache repo? You probably want to group those tests like for the backends.

Reply:
That's a good idea, I'll look into moving the tests into cache implementation repos when I add add more e2e tests with python `unittest` and clients, but not going to restructure it for 23.06.